packages/webapp/pages/squads/[handle]/index.tsx

```diff
@@ -123,18 +134,40 @@ const SquadPage = ({ handle }: SourcePageProps): ReactElement => {
     });
   }, [isForbidden, squadId, handle, trackEvent]);
 
-  if ((isFallback || isLoading) && !isFetched) return <SquadLoading />;
+  const seoData = squad || initialData;
+  const seo = !!seoData && (
+    <NextSeo
+      title={
+        referringUser
+          ? `${referringUser.name} invited you to ${seoData.name}`
+          : `${seoData.name} posts on daily.dev`
+      }
```

Comment:
We follow a specific template for titles. Have we confirmed this with the product team?

Reply:
Good point, probably good to keep it inline with the token page perhaps?
We also have something like `templatedTitle` that does the end part for you.

Reply:
@rebelchris @sshanzel It is in line with token page.

Reply:
Btw. looking at `getTemplatedTitle` why do we add `| daily.dev` to `titleTemplate` in `DefaultSeo` in `_app`? That would apply to all in one go.

Reply:
Hmm, not sure that's correct, maybe present the current ones to Sab and ask they still good.

As for templated, there are exclusions where we want it differently :)

Reply:
Confirmed with Daphna and Product that current titles are good. 👌

Reply:
I think we have it in DefaultSeo, can't remember exactly, but it is sometimes not working as intended.